src/transport/HighPressureGasTransport.cpp

```diff
@@ -327,167 +323,579 @@ double HighPressureGasTransport::viscosity()
+
+    // Molecular shapes and polarities factor, Equation 9-4.11
+    double Fc = 1 -0.2756*acentric_factor + 0.059035*pow(mu_r, 4.0) + kappa;
+    //cout << "Fc: " << Fc << ", Omega: " << omega << endl;
+
+    // Equation 9-3.9, multiplied by the Chung factor, Fc
+    // (another way of writing 9-4.10 that avoids explicit use of the critical volume in this method)
+    double viscosity = Fc* (26.69*sqrt(MW*T)/(sigma*sigma*omega));
+
+    return viscosity;
+}
+
+// Computes the high-pressure viscosity using the Chung method (Equation 9-6.18).
+// This function is structured such that it can be used for pure species or mixtures, with the
+// only difference being the values that are passed to the function (pure values versus mixture values).
+//
+// Renamed eta_star and eta_star_star from the Poling description to eta_1 and eta_2 for
+// naming simplicity.
+double ChungHighPressureGasTransport::high_pressure_viscosity(double T_star, double MW, double rho,
+                                                              double Vc, double Tc, double acentric_factor,
+                                                              double mu_r, double kappa)
+{
+    // Definition of tabulated coefficients for the Chung method, as shown in Table 9-6 on page 9.40 of Poling et al. (2001)
+    vector<double> a = {6.324, 1.210e-3, 5.283, 6.623, 19.745, -1.900, 24.275, 0.7972, -0.2382, 0.06863};
+    vector<double> b = {50.412, -1.154e-3, 254.209, 38.096, 7.630, -12.537, 3.450, 1.117, 0.06770, 0.3479};
+    vector<double> c = {-51.680, -6.257e-3, -168.48, -8.464, -14.354, 4.958, -11.291, 0.01235, -0.8163, 0.5926};
+    vector<double> d ={1189.0, 0.03728, 3898.0, 31.42, 31.53, -18.15, 69.35, -4.117, 4.025, -0.727};
+
+    // This is slightly pedantic, but this is done to have the naming convention in the
+    // equations used match the variable names in the code.
+    double E_1, E_2, E_3, E_4, E_5, E_6, E_7, E_8, E_9, E_10;
+    E_1 = a[0] + b[0]*acentric_factor + c[0]*pow(mu_r, 4.0) + d[0]*kappa;
+    E_2 = a[1] + b[1]*acentric_factor + c[1]*pow(mu_r, 4.0) + d[1]*kappa;
+    E_3 = a[2] + b[2]*acentric_factor + c[2]*pow(mu_r, 4.0) + d[2]*kappa;
+    E_4 = a[3] + b[3]*acentric_factor + c[3]*pow(mu_r, 4.0) + d[3]*kappa;
+    E_5 = a[4] + b[4]*acentric_factor + c[4]*pow(mu_r, 4.0) + d[4]*kappa;
+    E_6 = a[5] + b[5]*acentric_factor + c[5]*pow(mu_r, 4.0) + d[5]*kappa;
+    E_7 = a[6] + b[6]*acentric_factor + c[6]*pow(mu_r, 4.0) + d[6]*kappa;
+    E_8 = a[7] + b[7]*acentric_factor + c[7]*pow(mu_r, 4.0) + d[7]*kappa;
+    E_9 = a[8] + b[8]*acentric_factor + c[8]*pow(mu_r, 4.0) + d[8]*kappa;
+    E_10 = a[9] + b[9]*acentric_factor + c[9]*pow(mu_r, 4.0) + d[9]*kappa;
+
+    double y = rho*Vc/6.0; // Equation 9-6.20
+
+    double G_1 = (1.0 - 0.5*y)/(pow(1.0-y, 3.0)); // Equation 9-6.21
+
+    // Equation 9-6.22
+    double G_2 = (E_1*((1.0-exp(-E_4*y)) / y) + E_2*G_1*exp(E_5*y) + E_3*G_1)/ (E_1*E_4 + E_2 + E_3);
+
```

Comment:
At a minimum, keep spacing on each side of a binary operator the same:
```suggestion
    double G_2 = (E_1*((1.0-exp(-E_4*y)) / y) + E_2*G_1*exp(E_5*y) + E_3*G_1) / (E_1*E_4 + E_2 + E_3);

```